src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -3787,6 +3800,11 @@ inline void rpp_store24_i8pkd3_to_i8pln3_avx(Rpp8s* dstPtrR, Rpp8s* dstPtrG, Rpp
 }
 
 inline void rpp_store4_u8_to_u8(Rpp8u* dstPtr, __m128i &p)
```

Comment:
what is the use of 2 different functions below which is calling another inline function. Please call rpp_storeu_xx directly and remove these functions. Please don't add too many inline functions unless it is doing something useful